contracts/PriceConverter.sol

```diff
@@ -3,28 +3,20 @@ pragma solidity ^0.8.8;
 
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 
-// TODO: Typing club
 library PriceConverter {
-    //TODO: change priceconverter to a contract type
```

Comment:
I can't see a PR for this? I've created an issue to keep track of this: https://github.com/simkasss/hardhat-fund-me/issues/4